app/views/user_mailer/in_person_ready_to_verify.html.erb

```diff
@@ -0,0 +1,100 @@
+  <h2 class="margin-top-0 margin-bottom-2 font-heading-lg text-bold">
+    <%= t('in_person_proofing.body.barcode.items_to_bring') %>
+  </h2>
+  <p><%= t('in_person_proofing.body.barcode.emailed_info') %></p>
+  <table class="process-list">
+    <tr>
+      <td><div class="process-list__circle">1</div></td>
+      <td>
+        <h3 class="font-heading-md text-bold"><%= t('in_person_proofing.process.barcode.heading') %></h3>
+        <p><%= t('in_person_proofing.process.barcode.info') %></p>
+        <%= render BarcodeComponent.new(
+              barcode_data: @presenter.enrollment_code,
+              label: nil,
+              label_formatter: Idv::InPerson::EnrollmentCodeFormatter.method(:format),
+            ) %>
+      </td>
+    </tr>
+    <tr>
+      <td><div class="process-list__circle">2</div></td>
+      <td>
+        <h3 class="font-heading-md text-bold"><%= t('in_person_proofing.process.state_id.heading') %></h3>
+        <p class="margin-bottom-105"><%= t('in_person_proofing.process.state_id.info') %></p>
+        <ul class="usa-list margin-y-105">
+          <% t('in_person_proofing.process.state_id.acceptable_documents').each do |document| %>
+            <li><%= document %></li>
+          <% end %>
+        </ul>
+        <p><%= t('in_person_proofing.process.state_id.no_other_documents') %></p>
+      </td>
+    </tr>
+    <% if @presenter.needs_proof_of_address? %>
+      <tr>
+        <td><div class="process-list__circle">4</div></td>
+        <td>
+          <h3 class="font-heading-md text-bold"><%= t('in_person_proofing.process.proof_of_address.heading') %></h3>
+          <p class="margin-bottom-105"><%= t('in_person_proofing.process.proof_of_address.info') %></p>
+          <ul class="usa-list margin-y-105">
+            <% t('in_person_proofing.process.proof_of_address.acceptable_proof').each do |proof| %>
+              <li><%= proof %></li>
+            <% end %>
+          </ul>
+        </td>
+      </tr>
+    <% end %>
+  </table>
+  <p class="margin-bottom-0">
+    <%= t('in_person_proofing.body.barcode.items_to_bring_questions') %>
+    <%= link_to(
+          t('in_person_proofing.body.barcode.learn_more'),
+          MarketingSite.help_center_article_url(
+            category: 'verify-your-identity',
+            article: 'how-to-verify-in-person',
+          ),
+        ) %>
+  </p>
+</div>
+
+<div class="margin-y-4">
+  <h2 class="font-sans-md margin-bottom-1 text-normal text-bold"><%= @presenter.selected_location_details['name'] %></h2>
+  <div class="margin-bottom-1">
+    <%= @presenter.selected_location_details['streetAddress'] %><br>
+    <%= @presenter.selected_location_details['city'] %>,
+    <%= @presenter.selected_location_details['state'] %>
+    <%= @presenter.selected_location_details['zip5'] %>-<%= @presenter.selected_location_details['zip4'] %>
+  </div>
+  <h3 class="font-sans-sm margin-y-0 text-normal text-bold"><%= t('in_person_proofing.body.barcode.retail_hours') %></h3>
```

Comment:
Could we update "Retail hours" so that:
- the font size is the same as our body copy 
- bolded

Reply:
> Could we update "Retail hours" so that:
> 
> * the font size is the same as our body copy
> * bolded

@kellular Similar to my previous comment, this is also how it's designed in the Figma document. I can also change this one to use standard body font size.